go/src/koding/kites/gateway/gateway.go

```diff
@@ -0,0 +1,315 @@
+package gateway
```

Comment:
Can it be just `keygen`? In future it will handout tokens for any external service that klient uses if possible, e.g. for datadog or whatever will collect metrics on klient side.